### DISCUSSION TITLE
Readme: link version-label to crates.io

### DISCUSSION THREAD

**Discussion by ToBinio** [frewsxcv/rust-crates-index#138] [Ideas]
The current [![crates-index on Crates.io](https://img.shields.io/crates/v/crates-index.svg)](https://lib.rs/crates/crates-index) links to `lib.rs`. But I believe many expect it to link to `crates.io`. Which in my opinion would be more logical

note:
the same is true with the homepage link in the `cargo.toml`

**Comment by Byron** [comment]
I agree! Would you like to submit a PR for this? I'd appreciated it, thank you.

**Comment by ToBinio** [comment]
resolved in #140
